src/ci/resources/aws_application.conf

```diff
@@ -42,7 +42,7 @@ backend {
         concurrent-job-limit = 24
 
         default-runtime-attributes {
-          queueArn: "arn:aws:batch:us-east-1:952500931424:job-queue/highpriority-737f0580-c602-11ea-82b4-12cf2806950f"
+          queueArn: "arn:aws:batch:us-east-1:952500931424:job-queue/priority-cromwell-ci-stack-newest"
```

Comment:
TOL hopefully this will age well...

Reply:
There's always room for `priority-cromwell-ci-stack-even-newest-est-er`

Reply:
This is life science. We're used to docs being labeled as `Some Doc.latest.latest.latest.latest.docx`

Reply:
`next-nextgen`?

Reply:
Since I regularly delete old CF stacks, it allows me to go in circles with names:
`cromwel-ci-stack -> cromwell-ci-stack-new -> cromwell-ci-stack-newest -> cromwel-ci-stack -> ...`
🙂

Reply:
just always include a new anagram of chris' name for true uniqueness